test/quick_test/test_stats.py

```diff
@@ -0,0 +1,267 @@
+    else:
+        raise ValueError("Unknown stats file")
+
+
+def load_structs(test_file: Path):
+    """
+    Load the structs from the given file path.
+
+    Parameters
+    ----------
+    test_file : Path
+        Path to the test file.
+
+    Returns
+    -------
+    structs : list
+        List of structs.
+    """
+
+    if "aseg" in str(test_file):
+        structs_file = Path(__file__).parent / "data/thresholds/aseg.stats.yaml"
+    elif "aparc+DKT" in str(test_file):
+        structs_file = Path(__file__).parent / "data/thresholds/aparc+DKT.stats.yaml"
+    else:
+        raise ValueError("Unknown test file")
+
+    # Open the file_path and read the structs: into a list
+    with open(structs_file, "r") as file:
+        data = yaml.safe_load(file)
+        structs = data.get("structs", [])
+
+    return structs
+
+
+def read_measure_stats(file_path: Path):
+    """
+    Read the measure stats from the given file path.
+
+    Parameters
+    ----------
+    file_path : Path
+        Path to the stats file.
+
+    Returns
+    -------
+    measure : list
+        List of measures.
+    measurements : dict
+        Dictionary containing the measurements.
+    """
+
+    measure = []
+    measurements = {}
+
+    # Retrieve lines starting with "# Measure" from the stats file
+    with open(file_path, "r") as file:
+        # Read each line in the file
+        for i, line in enumerate(file, 1):
+            # Check if the line starts with "# ColHeaders"
+            if line.startswith("# ColHeaders"):
+                table_start = i
+                columns = line.strip("# ColHeaders").strip().split(" ")
+
+            # Check if the line starts with "# Measure"
+            if line.startswith("# Measure"):
+                # Strip "# Measure" from the line
+                line = line.strip("# Measure").strip()
+                # Append the measure to the list
+                line = line.split(", ")
+                measure.append(line[1])
+                measurements[line[1]] = float(line[3])
+
+    return measure, measurements
+
+
+def read_table(file_path: Path):
+    """
+    Read the table from the given file path.
+
+    Parameters
+    ----------
+    file_path : Path
+        Path to the stats file.
+
+    Returns
+    -------
+    table : pandas.DataFrame
+        Table containing the
+    """
+
+    table_start = 0
+    columns = []
+
+    file_path = file_path / "stats" / "aseg.stats"
+
+    # Retrieve stats table from the stats file
+    with open(file_path, "r") as file:
+        # Read each line in the file
+        for i, line in enumerate(file, 1):
+            # Check if the line starts with "# ColHeaders"
+            if line.startswith("# ColHeaders"):
+                table_start = i
+                columns = line.strip("# ColHeaders").strip().split(" ")
+
+    # Read the reference table into a pandas dataframe
+    table = pd.read_table(file_path, skiprows=table_start, sep="\s+", header=None)
```

Comment:
you can probably use `comment="#"` or something similar instead of this tablestart thing, but it is also fine as is.